src/environments/firebaseConfig.ts

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright 2020 Google LLC
```

Comment:
Can you add the full Apache license header here?

The `addlicense` tool is the easiest way to do this, [as described here](https://opensource.google/docs/releasing/preparing/#license-headers).

On a Unix-like system, you can automatically add any missing headers to TypeScript files with a shell command like:

`find . -name \*.ts | xargs addlicense -c "Google LLC" -l apache`

This shouldn't touch any files that already have the right header.